manim/mobject/functions.py

```diff
@@ -119,12 +119,12 @@ def generate_points(self):
 
 
 class FunctionGraph(ParametricFunction):
-    def __init__(self, function, color=YELLOW, **kwargs):
-        self.x_min = -config["frame_x_radius"]
-        self.x_max = config["frame_x_radius"]
+    def __init__(self, function, x_min=None, x_max=None, **kwargs):
+        self.x_min = x_min or -config["frame_x_radius"]
+        self.x_max = x_max or config["frame_x_radius"]
         self.parametric_function = lambda t: np.array([t, function(t), 0])
         ParametricFunction.__init__(
-            self, self.parametric_function, t_min=self.x_min, t_max=self.x_max, **kwargs
+            self, self.parametric_function, self.x_min, self.x_max, **kwargs
```

Comment:
Is there a reason to remove `t_min=` and such besides not being strictly necessary? Because if not then I'd prefer for them to be added back for readability's sake

Reply:
No specific reason, I was going back and forth between whether or not to add them. Since the function is directly passed as `self.parametric_function` and not `function=self.parametric_function`, I thought this should be fine. But if it's more readable to add `t_min=` and `t_max=`, I'll update that.